packages/offix-server-conflicts/src/api/ObjectConflictError.ts

```diff
@@ -0,0 +1,34 @@
+import { GraphQLError } from "graphql";
+import { ObjectStateData } from "./ObjectStateData";
+
+/**
+ * Data returned back to client wrapped in error objects.
+ * Client side link highly depend on this data to notify client about conflict
+ * and eventually perform client side conflict resolution
+ */
+export interface ConflictData {
+
+  /**
+   * Data that was modified on the server.
+   * Source of the conflict with additional fields that changed over the time.
+   */
+  serverState: ObjectStateData;
+
+  /**
+   * Original data that was sent from client and cannot be applied because of conflict
+   */
+  clientState: ObjectStateData;
+}
+
+/**
+ * Conflict error that is being returned when server
+ * Error specific to Voyager framework
+ */
+export class ObjectConflictError extends Error {
+  public conflictInfo: ConflictData;
+
+  constructor(data: ConflictData) {
+    super("VoyagerConflict");
```

Comment:
Still want the name `Voyager` in here?

Reply:
@Stephen WDYT about DataConflict?

Reply:
I do like the idea of having something like `VoyagerConflict` or `OffixConflict` or `OffixDataConflict` because it is a specific non-generic name for a specific thing that is slightly outside of the realm of regular GraphQL. As an end user that may not know things inside out, this at least makes it very clear in their application logs that they are dealing with something from offix. Helps lead them in the right direction quicker. WDYT?

Reply:
Yeah +1 to something specific. `OffixConflict` is good for me.